src/platform-includes/replay/privacy-configuration/javascript.mdx

```diff
@@ -1,4 +1,16 @@
-The following options can be configured as options to the integration, in `new Replay({})`:
+For a static website, free of personal identifiable or other type of private data.
```

Comment:
This recommendation placed on two docs pages feels quite aggressive and far from the privacy-first approach. :/ 

I wonder if we should still mark _inputs_ as masked in this default configuration. 

Also, I think we should talk about PII server-side scrubbing in place as it's additional insurance

Reply:
> This recommendation placed on two docs pages feels quite aggressive and far from the privacy-first approach. :/

We're still privacy-first. We need to be clear that there's the option and too often people come out confused that this is an option.

I could move it from the include to the _Configuration_ only page. Leaving the Privacy one without it? 
But since we have it in both I feel this fits well.

> I wonder if we should still mark inputs as masked in this default configuration.

You mean point out in this paragraph that user text will continue to be masked even with that? 

> Also, I think we should talk about PII server-side scrubbing in place as it's additional insurance

We talk about it somewhere else on the docs, do you mean also including it here?